code/harry_potter_store/README.md

```diff
@@ -0,0 +1,35 @@
+https://github.com/garora/TDD-Katas/blob/master/KatasReadme.md#harry-potter-
+
+To try and encourage more sales of the 5 different Harry
+Potter books they sell, a bookshop has decided to offer
+discounts of multiple-book purchases.
+
+One copy of any of the five books costs 8 EUR.
+
+If, however, you buy two different books, you get a 5%
+discount on those two books.
+
+If you buy 3 different books, you get a 10% discount.
+
+If you buy 4 different books, you get a 20% discount.
+
+If you go the whole hog, and buy all 5, you get a huge 25%
+discount.
+
+Note that if you buy, say, four books, of which 3 are
+different titles, you get a 10% discount on the 3 that
+form part of a set, but the fourth book still costs 8 EUR.
+
+Your mission is to write a piece of code to calculate the
+price of any conceivable shopping basket (containing only
+Harry Potter books), giving as big a discount as possible.
+
+For example, how much does this basket of books cost?
+
+2 copies of the first book
+2 copies of the second book
+2 copies of the third book
+1 copy of the fourth book
+1 copy of the fifth book
+
+Answer: 51.60 EUR
```

Comment:
I think this answer is not the right answer. At least I'm as a customer know a way how to buy this set of books for 51.2 and save myself one bus ride price =)

Would you be able to prove that greedy algorithm is correct here? Or do we need any addition to the problem statement to make it true?

Reply:
@okainov it is task description from link in top of README. 
Can you formulate your proposal more clearly?

Reply:
From the given task text I think this example is not correct. I don't know how much attention @kirill-korniakov  wants to put [on the algorithm part] here, but my proposal is to solve task correctly :) 

So either add missing conditions into task description and keep greedy algorithm or rework the solution so it gives really minimal/optimal solution.